frappe/tests/test_background_jobs.py

```diff
@@ -28,6 +28,22 @@ def test_remove_failed_jobs(self):
 				fail_registry = queue.failed_job_registry
 				self.assertEqual(fail_registry.count, 0)
 
+	def test_enqueue_at_front(self):
+		kwargs = {
+			"method": "frappe.handler.ping",
+			"queue": "short",
+		}
+
+		# give worker something to work on first so that get_position doesn't return None
+		frappe.enqueue(**kwargs)
```

Comment:
This is a hack, but it wasn't flaky in my testing. I tried using `rq.suspension.suspend`, but it didn't have the desired effect.